### PR TITLE
board/x86-multiboot: doxygen header fix

### PR DESCRIPTION
**Pull request by haukepetersen**

### CODE REVIEW

File: boards/x86-multiboot-common/startup.c [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this library; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
```
Comment (LudwigOrtmann) [inline boards/x86-multiboot-common/startup.c]: This isn't problematic.

Comment (haukepetersen) [inline boards/x86-multiboot-common/startup.c]: but the change doesn't hurt either, and it improves consistency

File: boards/x86-multiboot-common/include/multiboot.h [hunk 2]
```diff
@@ -16,11 +15,14 @@
    Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.  */
```
Comment (LudwigOrtmann) [inline boards/x86-multiboot-common/include/multiboot.h]: This license is problematic, but you can't change it ;)

File: boards/x86-multiboot-common/startup.c (hunk 1 above)
Comment (LudwigOrtmann) [inline boards/x86-multiboot-common/startup.c]: The license is pretty consistent..
```
$ wc -l dist/tools/licenses/out/lgplv2.1-v1
66 dist/tools/licenses/out/lgplv2.1-v1
```

File: boards/x86-multiboot-common/include/multiboot.h (hunk 2 above)
Comment (haukepetersen) [inline boards/x86-multiboot-common/include/multiboot.h]: but the license stayed the same, it't just referenced differently, isn't it?

Comment (LudwigOrtmann) [inline boards/x86-multiboot-common/include/multiboot.h]: You are confusing LGPL and GPL.

Comment (haukepetersen) [inline boards/x86-multiboot-common/include/multiboot.h]: oh dammit, that I overlooked... Will fix (revert) the change.

**Comment by haukepetersen** [comment]
reverted the license stuff.

**Comment by authmillenon** [comment]
ACK when Travis is happy.

**Comment by haukepetersen** [comment]
happy he is -> and go.

**Pull request merged**